Sources/TuistKit/Generator/LinkGenerator.swift

```diff
@@ -203,4 +215,43 @@ final class LinkGenerator: LinkGenerating {
             }
         }
     }
+    
+    func generateFakeSwiftDependenciesForStaticLibraries(
+        dependencies: [DependencyReference],
+        pbxTarget: PBXTarget,
+        pbxproj: PBXProj,
+        fileElements: ProjectFileElements
+    ) throws {
+
+        var files: [PBXBuildFile] = [ ]
+        
+        for case let .product(name) in dependencies {
+            
+            guard let fileRef = fileElements.product(name: name) else {
+                throw LinkGeneratorError.missingProduct(name: name)
+            }
+            
+            let buildFile = PBXBuildFile(file: fileRef)
+            pbxproj.add(object: buildFile)
+            files.append(buildFile)
+        }
+        
+        if files.count == 0 {
+            return
+        }
+        
+        let buildPhase = PBXCopyFilesBuildPhase(
+            dstPath: nil,
+            dstSubfolderSpec: .productsDirectory,
+            name: "[tuist] Fake Swift Dependencies",
```

Comment:
I'd avoid having references to `tuist` from the generated projects and files. What about calling the phase:

```
Transitive static dependencies
```

Reply:
I'm not sure "Transitive static dependencies" makes sense for the build phase name since they are actually the direct dependents of the library (not every single dependency).

maybe just "Dependencies" makes more sense here

Reply:
Renamed to `Dependencies`